pkg/kubectl/cmd/apply/apply.go

```diff
@@ -403,21 +404,21 @@ func (o *ApplyOptions) Run() error {
 			}
 
 			helper := resource.NewHelper(info.Client, info.Mapping)
-			patcher := &patcher{
-				mapping:       info.Mapping,
-				helper:        helper,
-				dynamicClient: o.DynamicClient,
-				overwrite:     o.Overwrite,
-				backOff:       clockwork.NewRealClock(),
-				force:         o.DeleteOptions.ForceDeletion,
-				cascade:       o.DeleteOptions.Cascade,
-				timeout:       o.DeleteOptions.Timeout,
-				gracePeriod:   o.DeleteOptions.GracePeriod,
-				serverDryRun:  o.ServerDryRun,
-				openapiSchema: openapiSchema,
+			patcher := &Patcher{
```

Comment:
The diff command uses this. Should we put a TODO here to refactor this into it's own file?